control.ipynb

```diff
@@ -0,0 +1,91 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0",
+   "metadata": {},
+   "source": [
+    "# AiiDA control"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ipywidgets as ipw\n",
+    "from aiida import load_profile\n",
+    "from IPython.display import clear_output, display\n",
+    "\n",
+    "load_profile()\n",
+    "from home.control import (\n",
+    "    DaemonControlWidget,\n",
+    "    GroupControlWidget,\n",
+    "    ProcessControlWidget,\n",
+    "    ProfileControlWidget,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "correspondance = {\n",
+    "    \"Daemon\": DaemonControlWidget,\n",
+    "    \"Group\": GroupControlWidget,\n",
+    "    \"Process\": ProcessControlWidget,\n",
+    "    \"Profile\": ProfileControlWidget,\n",
+    "}\n",
+    "\n",
+    "toc = ipw.ToggleButtons(\n",
+    "    options=correspondance.keys(),\n",
+    "    value=None,\n",
+    "    orientation=\"vertical\",\n",
+    "    layout=ipw.Layout(width=\"200px\"),\n",
+    ")\n",
+    "\n",
+    "\n",
+    "output = ipw.Output()\n",
+    "\n",
+    "\n",
+    "def update_output(value={\"new\": \"Group\"}):\n",
```

Comment:
I've merged in main with updated ruff that now checks jupyter notebooks. It reports a lint violation here:

```

control.ipynb:cell 3:19:25: B006 Do not use mutable data structures for argument defaults
   |
19 | def update_output(value={"new": "Group"}):
   |                         ^^^^^^^^^^^^^^^^ B006
20 |     if value["new"] in correspondance:
21 |         with output:
   |
   = help: Replace with `None`; initialize within function
```